src/vs/editor/browser/controller/textAreaHandler.ts

```diff
@@ -92,6 +93,7 @@ export class TextAreaHandler extends ViewPart {
 		const layoutInfo = options.get(EditorOption.layoutInfo);
 
 		this._accessibilitySupport = options.get(EditorOption.accessibilitySupport);
+		this._accessibilityLinesPerPage = options.get(EditorOption.accessibilityLinesPerPage);
```

Comment:
Please read this option also in the `onConfigurationChanged` method lower in the file. This would ensure that the setting can be changed dynamically at runtime.

Reply:
Yes. Makes sense.